SharedPackages/DesignResourcesKit/Sources/DesignResourcesKit/Colors/ColorPalletes/DefaultColorPalette.swift

```diff
@@ -111,6 +111,9 @@ struct DefaultColorPalette: ColorPaletteDefinition {
     private static let alertGreen = DynamicColor(lightColor: .alertGreen, darkColor: .alertGreen)
     private static let alertYellow = DynamicColor(lightColor: .alertYellow, darkColor: .alertYellow)
 
+    // Destructive
+    private static let destructivePrimary = DynamicColor(lightColor: Color(0xEE1025), darkColor: Color(0xFF545A))
```

Comment:
These colors here are defined as `alertRed` and `alertRed20`, let's use them here instead of hardcoded values.